server/routes/prisons/sessionTemplates/singleSessionTemplateController.ts

```diff
@@ -25,4 +25,40 @@ export default class SingleSessionTemplateController {
       })
     }
   }
+
+  public activate(): RequestHandler {
+    return async (req, res) => {
+      const { reference } = req.params
+
+      const sessionTemplate = await this.sessionTemplateService.activeSessionTemplate(
+        res.locals.user.username,
+        reference,
+      )
+      if (sessionTemplate.active) {
+        req.flash('message', 'activated')
```

Comment:
Setting a flash success message here (and in `deactivate()`) but not actually consuming it in the template.

To do so:
* add `message: req.flash('message')` to the `render` parameters in the `view()` method above
* copy a block like this (https://github.com/ministryofjustice/hmpps-visits-internal-admin-ui/blob/a8074d782b613c4275e07f0746c6529cc57530b1/server/views/pages/prisons/prisons.njk#L30) into the `viewSingleSessionTemplate.njk` at the same place. (We're duplicating this block of code a bit at the moment; Ben and I have discussed making this into a re-usable component as a future change)